napari_svg/_tests/test_write_points_with_attributes-expected.svg

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" standalone="no"?>
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg height="128" width="128" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g transform="translate(0 0)"><circle cx="0" cy="0" r="8.0" stroke="rgb(0, 255, 255)" fill="rgb(255, 0, 0)" stroke-width="0.8" opacity="0.5" /><circle cx="128" cy="0" r="10.0" stroke="rgb(255, 0, 255)" fill="rgb(0, 128, 0)" stroke-width="5.0" opacity="0.5" /><circle cx="128" cy="128" r="12.0" stroke="rgb(255, 255, 0)" fill="rgb(0, 0, 255)" stroke-width="12.0" opacity="0.5" /></g></svg>
\ No newline at end of file
+<svg height="128.0" width="128.0" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g transform="translate(-0.0 -0.0)"><circle cx="0" cy="0" r="8.0" stroke="rgb(0, 255, 255)" fill="rgb(255, 0, 0)" transform="matrix(1.0 0.0 0.0 1.0 0.0 0.0) translate(0.0 0.0) rotate(0.0) skewY(0.0) scale(1.0 1.0)" stroke-width="0.8" opacity="0.5" /><circle cx="128" cy="0" r="10.0" stroke="rgb(255, 0, 255)" fill="rgb(0, 128, 0)" transform="matrix(1.0 0.0 0.0 1.0 0.0 0.0) translate(0.0 0.0) rotate(0.0) skewY(0.0) scale(1.0 1.0)" stroke-width="5.0" opacity="0.5" /><circle cx="128" cy="128" r="12.0" stroke="rgb(255, 255, 0)" fill="rgb(0, 0, 255)" transform="matrix(1.0 0.0 0.0 1.0 0.0 0.0) translate(0.0 0.0) rotate(0.0) skewY(0.0) scale(1.0 1.0)" stroke-width="12.0" opacity="0.5" /></g></svg>
```

Comment:
Given there is a top-level group element (in `xml_to_svg`) and the transform for all the circles is the same, I would expect the transform to be defined once there and for each circle to have no transform.

Does that make sense? Or am I misunderstanding something about the SVG spec?

Doesn't necessarily block this PR, but might be worth doing if it's simple and makes the PR smaller.

Reply:
I just realized we are using that transform to shift everything so the view-box can start at 0, so maybe this isn't as easy as I'd hoped.

Reply:
yeah so actually you're both right and wrong... There should be one transform per layer. When I looked at the svg changes I thought "huh, I guess he put each point in a different layer. No idea why you would do that, maybe to test multiple layers, but ok." But now I look at the code and I see that there's one transform per shape!

The top level transform is indeed used to make a tight boundary around the data. But looking at the svg spec again it does look like I can nest things with `g` elements: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/g, so a layer should actually go in a g element and the transforms should go in there. I'll have a look shortly.

Reply:
After looking at the code a bit I'm not super confident about the updates, and they will definitely require updating the reference svg, so I'd like to experiment *after* this PR. I'm going to merge and create issues for the remaining work. Thanks for the review @andy-sweet!